conversions/Backwards/src/main/java/com/agentdid127/resourcepack/backwards/impl/PackMetaConverter.java

```diff
@@ -32,7 +32,12 @@ public void convert(Pack pack) throws IOException {
         Path file = pack.getWorkingPath().resolve("pack.mcmeta");
         if (!file.toFile().exists())
             return;
-        if (version >= Util.getVersionProtocol(packConverter.getGson(), "1.20"))
+        // Possible TODO: Make this JSON? Possibly use protocol.json, but update it.
+        if (version >= Util.getVersionProtocol(packConverter.getGson(), "1.20.2"))
+            versionInt = 22;
+        else if (version >= Util.getVersionProtocol(packConverter.getGson(), "1.20.2") && version < Util.getVersionProtocol(packConverter.getGson(), "1.20.3"))
```

Comment:
This should be 1.20.4. They are functionally the same, but it's nice to have it as a selection.